modules/administration-guide/pages/configuring-che-storage-size.adoc

```diff
@@ -0,0 +1,38 @@
+:_content-type: CONCEPT
+:description: Configuring the storage size in {prod-short}
+:keywords: administration guide, configuring, {prod-id-short}, storage size, PVC size, pvc
+:navtitle: Configuring the storage size in {prod-short}
+:page-aliases: installation-guide:configuring-che-storage-size.adoc
+
+[id="configuring-{prod-id-short}-workspace-pvc-size"]
+= Configuring the storage size in {prod-short}
+
+The persistent volume claim (PVC) size can be configured when using the `per-user` or `per-workspace` storage strategies. PVC sizes in the Che Cluster Custom Resource must be specified in the format of a https://kubernetes.io/docs/reference/kubernetes-api/common-definitions/quantity/[{kubernetes} resource quantity]. For more details on the available storage strategies, see xref:configuring-che-storage-strategy.adoc[]
+
+.Default persistent volume claim sizes:
+
+* `per-user`: `10Gi`
+* `per-workspace`: `5Gi`
+
+.Procedure
+
+Use CheCluster Custom Resource definition to define the persistent volume claim size for the desired storage strategy:
```

Comment:
This appears to belong in the intro paragraph or two that currently starts on line 10.

Reply:
For now, I'm going to remove this sentence as it doesn't seem to add any extra information that the first paragraph doesn't already have.